css/components/header/in-site/list.css

```diff
@@ -0,0 +1,33 @@
+.header__in-site__list {
+    display: none;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+}
+
+.header__in-site__list__item {
+    margin: 0 20px;
+    list-style: none;
+}
+
+.header__in-site__list__item a {
+    height: 52px;
+    display: inline-flex;
+    align-items: center;
+}
+
+@media screen and (min-width: 360px) {
+    .header__in-site__list {
+        display: flex;
+    }
+}
+
+.header__in-site__icon:focus-within + .header__in-site__list {
+    display: block;
+    position: absolute;
+    max-width: 120px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    margin-top: 40px;
+    z-index: 1
```

Comment:
my advice here, try not to use z-index only if nothing else works

Reply:
I remove it, and it continue working, Thanks for the advice